server/bin/pbench-verify-indexed.py

```diff
@@ -123,6 +123,17 @@ def main(options):
         indexed[tb_file] += 1
     indexed_cnt = len(indexed)
 
+    # Look for duplicates in the returned Elasticsearch data to help detect
+    # problems with the indexer.
+    duplicates = []
+    for tb_file, val in indexed.items():
+        if val > 1:
+            duplicates.append(tb_file)
```

Comment:
`duplicates = [n for n, v in indexed.items() if v > 1]` ???

The comprehension is not only more compact, I think it's easier to read. (Even if you use `name, val`, although I think single-character variables are usually pretty effective for small blocks like this.)